keras_cv/layers/preprocessing/random_flip_test.py

```diff
@@ -36,6 +36,13 @@ def test_horizontal_flip(self):
             actual_output = layer(inp, training=True)
             self.assertAllClose(expected_output, actual_output)
 
+    def test_flip_ragged(self):
+        images = tf.ragged.stack([tf.ones((512, 512, 3)), tf.ones((1002, 512, 3))])
+        bounding_boxes = tf.ragged.stack([tf.ones((5, 5)), tf.ones((3, 5))])
```

Comment:
Do you think it is useful to test a different number of bboxes x image. E.g. 1 for the 1st image and 2 for the 2nd one?

Reply:
That’s what this does

Reply:
Oh ok,  3 vs 5.